services/tenant-ui/frontend/src/store/verifierStore.ts

```diff
@@ -1,10 +1,13 @@
 import { defineStore } from 'pinia';
 import { ref } from 'vue';
-import { fetchList } from './utils/fetchList.js';
+import { fetchList } from './utils/fetchList';
+import { fetchItem } from './utils/fetchItem';
+
 
 export const useVerifierStore = defineStore('verifier', () => {
   // state
   const presentations: any = ref(null);
+  const presentationDetailDict: object = ref({});
```

Comment:
see other comments... i don't think this is a good pattern to adopt. calling components can manage whatever data they need on demand. in this PR, if they expand 20 different presentations, the presentations view can manage that collection.